src/ausweiskopie/editor/exporter.py

```diff
@@ -0,0 +1,115 @@
+import json
+from typing import Optional, Collection, Tuple, Mapping
+
+from ausweiskopie.editor.model import FieldLocation
+from ausweiskopie.redact import Location, Field
+
+
+class BaseExporter:
```

Comment:
I think making this an AbstractBaseClass (ABC) would be a good idea.

Reply:
I did not know, Python had such a thing 😮
Guess my Python foo is a little rusty. Will rework accordingly.